tdrs-backend/tdpservice/search_indexes/test/test_model_mapping.py

```diff
@@ -329,30 +355,33 @@ def test_can_create_and_index_tanf_t6_submission():
 
 
 @pytest.mark.django_db
-def test_can_create_and_index_tanf_t7_submission():
+def test_can_create_and_index_tanf_t7_submission(test_datafile):
     """TANF T7 submissions can be created and mapped."""
     record_num = fake.uuid4()
 
-    submission = models.tanf.TANF_T7.objects.create(
-        record=record_num,
-        rpt_month_year=1,
-        fips_code='2',
-
-        calendar_quarter=1,
-        tdrs_section_ind='1',
-        stratum='1',
-        families=1
-    )
-
-    assert submission.id is not None
-
-    search = documents.tanf.TANF_T7DataSubmissionDocument.search().query(
-        'match',
-        record=record_num
-    )
-    response = search.execute()
-
-    assert response.hits.total.value == 1
+    submission = models.tanf.TANF_T7()
+    submission.version = test_datafile.version
+    submission.created_at = test_datafile.created_at
+    submission.datafile = test_datafile
+    submission.record = record_num
+    submission.rpt_month_year = 1
+    submission.fips_code = '2'
+    submission.calendar_quarter = 1
+    submission.tdrs_section_ind = '1'
+    submission.stratum = '1'
+    submission.families = 1
+
+    submission.save()
+
+    # assert submission.id is not None
```

Comment:
Can you add in why this is commented out? Something like "t7 not yet implemented/parsed" is fine.